views/upload.njk

```diff
@@ -0,0 +1,24 @@
+{% extends '_user.njk' %}
+{% block css %}
+<link href="/stylesheets/style.css" rel="stylesheet">
+{% endblock %}
+{% block content %}
+{{super() }}
+<div class="row" style="text-align: center;">     
+  <div class="span12">
+    <h1>Upload file here!</h1>         
+    <div class="centerupload">
+     <form action="/upload" id="form1">
+      <input type='file' id="imgInp" accept="file"/*>
+      <br></br>
+      <img id="file" src="#"/>
+      <br></br>
+      <button class="btn btn-primary" role="button">Upload</button>
+     </form>
+    </div>
+  </div>
+</div>
+{% endblock %}
+{% block script%}
+<script src="/js/upload.js"></script>
```

Comment:
`/js/upload.js` does not seem to exist, I am getting errors because it cannot retrieve that file. Do you plan on adding it later? If it is required please add it as part of this branch and PR.